doc/indices.sgm

```diff
@@ -100,7 +112,6 @@
 <![CDATA[CREATE INDEX test_pos_idx USING BRIN ON test (pos) WITH (pages_per_range = 16);]]>
           </programlisting>
         </example>
-
```

Comment:
Based on the other `</sect1>` tags I saw, I don't this blank line before `</sect1>` should have been removed.